lib/measured/conversion_table_builder.rb

```diff
@@ -1,14 +1,29 @@
 class Measured::ConversionTableBuilder
   attr_reader :units
 
-  def initialize(units)
+  def initialize(units, cache: nil)
     @units = units
+    cache ||= { class: Measured::Cache::Null }
```

Comment:
Nitpick: It would be nice to rename this. Right now we have both `cache` and `@cache`, which is a little bit confusing imo. Maybe rename the local variable to something like `cache_settings` or `cache_options`?

Reply:
Renaming it changes the signature of the method. And I want it to be declared with the `cache:` kwarg such as:

```ruby
Measured::ConversionTableBuilder.new(units, cache: { class: MyCache })

Reply:
It's only confusing for one line. I can refactor it if you'd like. I don't think this is unreadable though.

Reply:
It's fine to keep `cache`, it's just a nitpick of mine. One question though: Shouldn't you merge the defaults instead of using `||=`? If whoever is calling do not pass a `class`, things could blow up.

Reply:
I'm not worried about this. Again it's only called by the `cache` line in the DSL, which has the syntax of:

```ruby
cache CacheClass, *options
```

So it'll be passed in in that format.

Reply:
I think it might still be worth merging, since someone reading this code may not immediately know its only called in one place, and it _could_ be confusing. Not a blocker by any means though